uniffi_bindgen/src/interface/object.rs

```diff
@@ -327,6 +332,10 @@ impl Method {
         self.attributes.get_throws_err()
     }
 
+    pub fn self_by_arc(&self) -> bool {
```

Comment:
The parsing center of my brain wants the name of this to be worded as a predicate about the object, e.g. `takes_self_by_arc` or similar. But then I can't really articulate a cohesive style guide for the way things are named in here, so I'm just going to note it and move on.